qiskit_nature/second_q/operators/second_quantized_op.py

```diff
@@ -71,7 +71,25 @@ def is_hermitian(self, atol: Optional[float] = None) -> bool:
         Returns:
             True if the operator is hermitian up to numerical tolerance, False otherwise.
         """
+        return self.equiv(self.adjoint(), atol=atol)
+
+    def equiv(self, other: Any, atol: Optional[float] = None) -> bool:
+        """Checks whether this operator is approximately equal to another operator.
```

Comment:
Please also add a comment here (similar to the `induced_norm` one) that this method does not normal-order or simplify either side of the comparison, since, again, doing that could affect the outcome of this comparison

Reply:
Done (note the method actually does simplify).